CODING_STANDARDS.md

```diff
@@ -0,0 +1,263 @@
+# Coding Standards
+
+This is a working document outlining the coding standard for the general MariaDB codebase.
+It does not cover the coding standards for individual plugins, these should have their own coding standards documentation.
+
+## Using Git with the MariaDB codebase
+
+### Git commit messages
+
+Git commit messages must conform to the 50/72 rule.
+This is is a long standard git standard which is automatically enforced by some editors.
```

Comment:
grammar: "long standard git standard"

Perhaps intended to say "long standing git standard"?

Also I would say "de facto standard" or replace standard with "convention".

Reply:
I think I meant "long standing git standard". But I'll rephrase.